config_serviceflags.go

```diff
@@ -31,6 +31,7 @@ type ServiceFlags struct {
 	BackgroundReauthDisabled      bool `bson:"background_reauth_disabled" json:"background_reauth_disabled"`
 	BackgroundCleanupDisabled     bool `bson:"background_cleanup_disabled" json:"background_cleanup_disabled"`
 	AmboyRemoteManagementDisabled bool `bson:"amboy_remote_management_disabled" json:"amboy_remote_management_disabled"`
+	ProjectsTrackBranchesDisabled bool `bson:"projects_track_branches_disabled" json:"projects_track_branches_disabled"`
```

Comment:
What will this be used for? Is it more a feature flag or something that we might need to disable once the project is done?

Reply:
Also resolved in person I think.